Extensions/BBText/pixi-multistyle-text/README.md

```diff
@@ -0,0 +1,7 @@
+This extension is using a forked version of https://github.com/tleunen/pixi-multistyle-text, which adds the ability to handle bbcode styled tags.
```

Comment:
Add a link to the commit/branch on GitHub of this fork so that anyone can fetch it and rebuild it, for example if there is an urgent bugfix to do.